org.adempiere.base/src/org/compiere/model/MRole.java

```diff
@@ -124,7 +124,7 @@ private static MRole getDefaultRole() {
 	 */
 	public synchronized static MRole get (Properties ctx, int AD_Role_ID, int AD_User_ID, boolean reload)
 	{
-		if (s_log.isLoggable(Level.INFO)) s_log.info("AD_Role_ID=" + AD_Role_ID + ", AD_User_ID=" + AD_User_ID + ", reload=" + reload);
+		if (s_log.isLoggable(Level.CONFIG)) s_log.info("AD_Role_ID=" + AD_Role_ID + ", AD_User_ID=" + AD_User_ID + ", reload=" + reload);
```

Comment:
this doesn't looks correct, the s_log call is using info level.

Reply:
thanks - fixed